docusaurus.config.js

```diff
@@ -134,8 +134,8 @@ module.exports = {
             type: 'all',
             copyright,
           },
-          blogSidebarTitle: 'All posts',
-          blogSidebarCount: 'ALL',
+          blogSidebarTitle: 'Recent posts',
+          blogSidebarCount: 5,
```

Comment:
Any reason to not show all blogs on the sidebar?

Reply:
No particular reason. I thought about showing only the latest 5 to reduce the list items in the sidebar.